tox.ini

```diff
@@ -6,9 +6,9 @@ usedevelop=True
 
 [testenv:py{39,310,311,312}]
 description = Run the test suite against Python versions
-allowlist_externals = poetry
+allowlist_externals = poetry, sh
 commands_pre = poetry install --no-root --sync
-commands = poetry run pytest --cov aiida_mlip --import-mode importlib
+commands = sh -c 'export JANUS_PATH=$(which janus); exec "$@"' _ poetry run pytest --cov aiida_mlip --import-mode importlib
```

Comment:
A comment explaining the shell magic here might be nice.

Reply:
@ElliottKasoar can you help in here